sdk/core/azure-core/src/main/java/com/azure/core/cryptography/IKeyEncryptionKey.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.cryptography;
+
+import reactor.core.publisher.Mono;
+
+
+/**
+ * A key which is used to encrypt, or wrap, another key.
+ */
+public interface IKeyEncryptionKey {
+    
+    /**
```

Comment:
Could there be a getKeyId() method? This was also a change that was made in .NET and is required for a preview 4 storage feature

Reply:
Added it